molgroups/refl1d_interface/plots.py

```diff
@@ -200,9 +197,18 @@ def cvo_uncertainty_plot(layer: MolgroupsLayer, model: Experiment | None = None,
     points = points[-n_samples:-1]
     #print('\n'.join(['%i\t%s' % a for a in enumerate(state.labels)]))
 
-    mapper = MPMapper.start_mapper(problem)
-    results = MPMapper.pool.map(_MP_calc_profile, ((MPMapper.problem_id, list(problem.models).index(model), pt) for pt in points))
-    MPMapper.stop_mapper(mapper)
+    # set up a parallel calculation
+    model_index = list(problem.models).index(model)
+    serialized_problem_array = array.array('B', dill.dumps(problem))
+
+    with multiprocessing.Manager() as manager:
+        shared_serialized_problem = manager.Array("B", serialized_problem_array)
```

Comment:
I think you can just do `shared_serialized_problem = manager.Array("B", dill.dumps(problem))` but I think creating the array.array doesn't cost you anything either.